build_conda_arm64.sh

```diff
@@ -4,7 +4,7 @@ cd $PYSAMDIR || exit
 source $(conda info --base)/etc/profile.d/conda.sh
 
 # get `CONDA_TOKEN` env variable, required for authentication
-source ~/.env || exit
+#source ~/.env || exit
```

Comment:
was this also broken / causing problems for you?

Reply:
> was this also broken / causing problems for you?

It was but I updated the .env file and added $ROOTDIR export to not be tied to sam-dev folder.

Reply:
I see, should it still be commented out?

Reply:
> I see, should it still be commented out?

Uncommented in latest commit. Thanks!

Reply:
Thanks!